SSA/Core/ErasedContext.lean

```diff
@@ -56,6 +56,16 @@ def get? : Ctxt Ty → Nat → Option Ty :=
 def map (f : Ty₁ → Ty₂) : Ctxt Ty₁ → Ctxt Ty₂ :=
   List.map f
 
+@[simp]
+lemma map_nil (f : Ty → Ty') :
+  map f ∅ = ∅ := rfl
+
+@[simp]
+lemma map_cons (Γ : Ctxt Ty) (t : Ty) (f : Ty → Ty') :
+  map f (Γ.cons t) = (Γ.map f).cons (f t) := rfl
```

Comment:
There's some inconsistent use of dot-notation here (`map f _` vs `_.map f`).

Also, I think `cons` here is referring to `List.cons`, which we've alliased as `Ctxt.snoc`.
In the spirit of remaining higher-level, I think we should not be adding lemmas for `List.snoc`, to nudge us towards not unfolding `Ctxt.snoc` in the first place